features/raadpleeg-verblijfplaats-met-periode/gba/overzicht-gba.feature

```diff
@@ -0,0 +1,330 @@
+    Gegeven de persoon met burgerservicenummer '000000024' is ingeschreven op adres 'A1' met de volgende gegevens
+    | gemeente van inschrijving (09.10) | datum aanvang adreshouding (10.30) |
+    | 0800                              | 20100600                           |
+    En de persoon is vervolgens ingeschreven op adres 'A4' met de volgende gegevens
+    | gemeente van inschrijving (09.10) | datum aanvang adreshouding (10.30) |
+    | 0800                              | 20100000                           |
+    En de persoon is vervolgens ingeschreven op een buitenlands adres met de volgende gegevens
+    | datum aanvang adres buitenland (13.20) | land adres buitenland (13.10) |
+    | 20101026                               | 6014                          |
+    En de persoon is vervolgens ingeschreven op adres 'A2' met de volgende gegevens
+    | gemeente van inschrijving (09.10) | datum aanvang adreshouding (10.30) |
+    | 0800                              | 20200415                           |
+    Als gba verblijfplaatsen wordt gezocht met de volgende parameters
+    | naam                | waarde              |
+    | type                | RaadpleegMetPeriode |
+    | burgerservicenummer | 000000024           |
+    | datumVan            | 2010-07-01          |
+    | datumTot            | 2020-07-01          |
+    Dan heeft de response verblijfplaatsen met de volgende gegevens
+    | land.code | land.omschrijving            | straat      | adresseerbaarObjectIdentificatie | locatiebeschrijving         | datumAanvangAdreshouding | datumAanvangVolgendeAdreshouding | datumAanvangAdresBuitenland | datumAanvangVolgendeAdresBuitenland |
+    |           |                              | Luttestraat | 0800010000000002                 |                             | 20200415                 |                                  |                             |                                     |
+    | 6014      | Verenigde Staten van Amerika |             |                                  |                             |                          | 20200415                         | 20101026                    |                                     |
+    |           |                              |             |                                  | Woonboot bij de Grote Sloot | 20100000                 |                                  |                             | 20101026                            |
+    |           |                              | Laan        | 0800010000000001                 |                             | 20100600                 | 20100000                         |                             |                                     |
+
+Rule: Een verblijfplaats van een persoon met geheel onbekende datum aanvang adreshouding/adres buitenland wordt altijd geleverd
```

Comment:
Ik ben het niet eens met deze regel. Resultaat is dat de gebruiker geen duidelijk beeld meer heeft van welke verblijfplaatsen er waren binnen de gevraagde periode. 
Daarvoor moet er m.i., net als in de bewoning API, rekening worden gehouden met vorige (bij onbekende aanvang) en daaropvolgende (bij onbekende aanvang volgende).

Reply:
aanvullende tekst en scenarios zijn toegevoegd om te illustreren hoe in deze gevallen wordt bepaald welke verblijfplaats voorkomens wordt geleverd